### ISSUE TITLE
riff system install --latest

### ISSUE THREAD

**Issue by jldec** [projectriff/riff#686]
i know there's some complexity because of riff system install currently using reduced/lite or broken-up versions of the install yaml, but it would be nice to have an option to install latest using something like this:

install master istio
```
kubectl apply -f https://raw.githubusercontent.com/knative/serving/master/third_party/istio-1.0.0/istio.yaml
kubectl label namespace default istio-injection=enabled
```
install latest knative serving
```
kubectl apply -f https://storage.googleapis.com/knative-releases/serving/latest/release.yaml
```
install latest knative eventing and stub bus
```
kubectl apply -f https://storage.googleapis.com/knative-releases/eventing/latest/release.yaml
kubectl apply -f https://storage.googleapis.com/knative-releases/eventing/latest/release-clusterbus-stub.yaml
```

**Comment by jldec** [comment]
feel free to close if already addressed by #670

**Comment by trisberg** [comment]
This can now be done with a manifest like:

```
version: 0.1
istio:
- https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml
knative:
- https://storage.googleapis.com/knative-releases/serving/latest/release-no-mon.yaml
- https://storage.googleapis.com/knative-releases/eventing/latest/release.yaml
- https://storage.googleapis.com/knative-releases/eventing/latest/release-clusterbus-stub.yaml
```

**Comment by trisberg** [comment]
Resolved via template feature in #691
